ci-operator/step-registry/redhat-chaos/krkn-operator/install/redhat-chaos-krkn-operator-install-commands.sh

```
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -o errexit
+
+set -o nounset
+set -o pipefail
+set -x
+
+
+helm install krkn-operator oci://quay.io/krkn-chaos/charts/krkn-operator \
+  --version $KRKN_OPERATOR_VERSION \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)